sympy/concrete/summations.py

```diff
@@ -988,6 +988,24 @@ def eval_sum_symbolic(f, limits):
 
         r = gosper_sum(f, (i, a, b))
 
+        # to find Sum at points where denominator approaches to zero
+        from sympy import together, fraction, Mul, Add, ordered
+        if isinstance(r, (Mul,Add)) and not(any(x in r.atoms(Symbol) for x in limits)):
+            var = ordered(r.atoms(Symbol))
+            gar, denom = fraction(together(r))
```

Comment:
It doesn't appear that you use `gar` later -- using underscore for such throw-aways is an idiom we use (`_, denom = ...`).